kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RuntimeDataServiceIntegrationTest.java

```diff
@@ -1188,13 +1188,13 @@ public void testFindTasksByStatusByProcessInstanceId() throws Exception {
             TaskSummary taskInstance = tasks.get(0);
```

Comment:
This checking of task instance could be also introduced as a method to avoid duplication.